packages/admin/admin/src/mui/menu/CollapsibleItem.tsx

```diff
@@ -1,43 +1,28 @@
-import { ChevronDown, ChevronUp } from "@comet/admin-icons";
-import { Collapse, ComponentsOverrides, List, Theme } from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ChevronDown, ChevronRight, ChevronUp } from "@comet/admin-icons";
+import { Collapse, ComponentsOverrides, Fade, List, Menu, Theme, Typography } from "@mui/material";
+import { MenuProps } from "@mui/material/Menu/Menu";
+import { SvgIconProps } from "@mui/material/SvgIcon";
+import { WithStyles, withStyles } from "@mui/styles";
 import * as React from "react";
 import { matchPath, useLocation } from "react-router";
 
+import { MenuCollapsibleItemClassKey, styles } from "./CollapsibleItem.styles";
 import { MenuItem, MenuItemProps } from "./Item";
 import { MenuItemRouterLinkProps } from "./ItemRouterLink";
 
-export type MenuCollapsibleItemClassKey = "root" | "childSelected" | "listItem" | "open";
-
-const styles = (theme: Theme) =>
-    createStyles<MenuCollapsibleItemClassKey, MenuCollapsibleItemProps>({
-        root: {},
-        childSelected: {
-            color: theme.palette.primary.main,
-            "& $listItem": {
-                "& [class*='MuiListItemText-root']": {
-                    color: theme.palette.primary.main,
-                },
-                "& [class*='MuiListItemIcon-root']": {
-                    color: theme.palette.primary.main,
-                },
-            },
-        },
-        listItem: {},
-        open: {},
-    });
-
 export interface MenuLevel {
     level?: 1 | 2 | 3;
 }
 
-type MenuChild = React.ReactElement<MenuItemRouterLinkProps>;
+export type MenuChild = React.ReactElement<MenuCollapsibleItemProps | MenuItemRouterLinkProps | MenuItemProps>;
 
 export interface MenuCollapsibleItemProps extends MenuItemProps {
     children: MenuChild | MenuChild[];
     openByDefault?: boolean;
-    openedIcon?: React.ReactNode;
-    closedIcon?: React.ReactNode;
+    OpenedIcon?: React.JSXElementConstructor<SvgIconProps>;
+    ClosedIcon?: React.JSXElementConstructor<SvgIconProps>;
```

Comment:
Why change this? It is a breaking change. Also, we use lower case prop names for other React components...

Reply:
You're right I changed it back